openpype/hosts/maya/api/plugin.py

```diff
@@ -234,11 +235,17 @@ def update(self, container, representation):
             path = self.prepare_root_value(path,
                                            representation["context"]
                                                          ["project"]
-                                                         ["code"])
+                                                         ["name"])
```

Comment:
Also, this change got in there by accident, or?

Reply:
That was actually a bug! `prepare_root_value` takes the project name, not the code!